cpp/src/Ice/PropertyUtil.h

```diff
@@ -14,6 +16,18 @@ namespace IceInternal
     /// @param propertyArray The property array to search.
     /// @return The property if found, nullopt otherwise.
     std::optional<Property> findProperty(std::string_view key, const PropertyArray* propertyArray);
+
+    /// Finds all unknown properties for a given prefix.
+    /// @param propertyClass The property class name.
```

Comment:
I think propertyClass should be the last parameter, not the first one.

Reply:
Or better, add it to propertyArray.

Reply:
Agree, given the method name it makes more sense to have `prefix` as the first parameter.